Dockerfile

```
@@ -1,33 +1,44 @@
+    && rm -rf /var/lib/apt/lists/*
+
+COPY --from=builder /app/.next/standalone ./
+COPY --from=builder /app/.next/static ./.next/static
+COPY --from=builder /app/public ./public
+
 EXPOSE 3000
 
-# migrate real db, then start
-# using 'pnpm start' because we have the full environment now
-CMD pnpm prisma migrate deploy && pnpm start
+CMD node server.js
```

Hadolint (2.14.0)

[warning] Use arguments JSON notation for CMD and ENTRYPOINT arguments

(DL3025)